apps/web/pages/getting-started.tsx

```diff
@@ -352,25 +352,6 @@ export default function Onboarding(props: inferSSRProps<typeof getServerSideProp
           </div>
           <form className="sm:mx-auto sm:w-full">
             <section className="space-y-8">
-              {(props.usernameParam || props.user?.identityProvider !== IdentityProvider.CAL) && (
```

Comment:
This form value was removed, since its no longer required as you should now pickup a valid username at signup